rs/sns/swap/proto/ic_sns_swap/pb/v1/swap.proto

```diff
@@ -517,10 +519,15 @@ message CfNeuron {
 
 // Represents a Neurons' Fund participant, possibly with several neurons.
 message CfParticipant {
-  // The principal that can vote on behalf of these Neurons' Fund neurons.
-  string hotkey_principal = 1;
+  // The principal that can manage this neuron.
```

Comment:
It would be clearer to say managing voting permissions. It shouldn't have full control.

Reply:
It is very confusing in this case because `// The principal that can vote on behalf of these Neurons' Fund neurons.` in the original comment was referring to the SNS while `// The principal that can manage this neuron.` in the new comment is referring to the NNS. I've clarified the comment to prevent this confusion.